src/lib/pci_virtio_rnd.c

```diff
@@ -53,8 +53,8 @@
 
 
 static int pci_vtrnd_debug;
-#define DPRINTF(params) if (pci_vtrnd_debug) printf params
```

Comment:
there shouldn't be any need to change this debug code. this won't be enabled for release builds and there should be no need to redirect to a logger.